design/cluster-scope-resource-filter.md

```diff
@@ -0,0 +1,502 @@
+	--include-cluster-scope-resources=*
+	--include-cluster-resources=true
+```
+
+This means backup Deployment, Service, Endpoint, Pod and ReplicaSet resources in all namespaces, and all cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=*
+```
+
+This means backup Deployment, Service, Endpoint, Pod and ReplicaSet resources in default and kube-system namespaces, and all cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=default,kube-system
+	--include-namespaced-resources=Deployment,Service,Endpoint,Pod,ReplicaSet
+	--include-cluster-resources=true
+	--include-cluster-scope-resources=*
+```
+
+#### all namespaced resources + no cluster resources
+The following commands all mean backup all namespace scope resources and no cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=*
+	--include-namespaced-resources=*
+	--include-cluster-resources=false
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=*
+	--include-cluster-resources=false
+```
+
+#### all namespaced resources + only related cluster resources
+The following commands all mean backup all resources in all namespaces, and related cluster resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=*
+	--include-namespaced-resources=*
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=*
+```
+
+#### all namespaced resources + some additional cluster resources
+The following commands all mean backup all namespace scope resources, and related cluster resources, plus all PersistentVolume resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=*
+	--include-cluster-scope-resources=PersistentVolume
+	--include-cluster-resources=true
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=*
+	--include-cluster-scope-resources=PersistentVolume
+	--include-cluster-resources=true
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=*
+	--include-namespaced-resources=*
+	--include-cluster-scope-resources=PersistentVolume
+	--include-cluster-resources=true
+```
+
+#### all namespaced resources + all cluster resources
+The following commands have the same meaning: backup all namespace scope resources, and all cluster scope resources.
+``` bash
+velero backup create <backup-name>
+	--include-namespaces=*
+	--include-namespaced-resources=*
+	--include-cluster-scope-resources=*
+	--include-cluster-resources=true
+```
+
+``` bash
+velero backup create <backup-name>
+	--include-namespaced-resources=*
+	--include-cluster-scope-resources=*
+	--include-cluster-resources=true
+```
+
+#### describe command change
+In `velero backup describe` command, the four new parameters should be outputted too.
```

Comment:
If some the of parameters are mutual exclusive, we should print the new ones only, otherwise it will confuse the users.